packages/core/upload/admin/src/pages/App/index.js

```diff
@@ -1,31 +1,122 @@
 import React from 'react';
-import { Switch, Route, Redirect } from 'react-router-dom';
-import { LoadingIndicatorPage, useRBAC } from '@strapi/helper-plugin';
-import pluginId from '../../pluginId';
+import { Redirect } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import {
+  LoadingIndicatorPage,
+  useRBAC,
+  useFocusWhenNavigate,
+  NoPermissions,
+  NoMedia,
+  AnErrorOccurred,
+} from '@strapi/helper-plugin';
+import { Layout, HeaderLayout, ContentLayout, ActionLayout } from '@strapi/parts/Layout';
+import { Main } from '@strapi/parts/Main';
+import { Button } from '@strapi/parts/Button';
+import AddIcon from '@strapi/icons/AddIcon';
+import { Box } from '@strapi/parts/Box';
+import { BaseCheckbox } from '@strapi/parts/BaseCheckbox';
+import { Search } from './components/Search';
```

Comment:
You can use the Search component from the helper plugin instead, it is not wrapped in the SearchForm one but you can make an update if needed.

Reply:
Also what since the timestamps are not configurable with the v4, what do you think of settings the url's parameters directly so you won't have to handle any logic in your child components:

``` 
const [{ query }, setQuery] = useQueryParams();

  React.useEffect(() => {
    if (!query) {
      setQuery({ sort: 'updatedAt:DESC', page: 1, pageSize: 10 });
    }
  }, [query, setQuery]);
```